test_app/tests/oauth2_provider/test_models.py

```diff
@@ -32,3 +36,79 @@ def test_oauth2_revoke_refresh_token(oauth2_admin_access_token):
     new_refresh_token = OAuth2RefreshToken.objects.all().first()
     assert refresh_token == new_refresh_token
     assert new_refresh_token.revoked
+
+
+@pytest.mark.django_db
+@mock.patch("ansible_base.oauth2_provider.models.access_token.logger")
+def test_oauth2_access_token_creation_logs_with_application(mock_logger, oauth2_application_password, admin_user):
+    """Test that OAuth2AccessToken creation logs with application name."""
+    application, _secret = oauth2_application_password
+
+    # Create an access token with an application
+    token = OAuth2AccessToken.objects.create(
+        user=admin_user,
+        application=application,
+        token=generate_token(),
+        scope='write',
+        expires=datetime(2088, 1, 1, tzinfo=timezone.utc),
+    )
+
+    # Verify the logger was called with the correct message
```

Comment:
The test mocks the module logger and expects `.info()` to be called, but `log_auth_event` uses `.log(level, message)` not `.info(message)`. These are different method signatures.

If this test passes, either:
1. The `log_auth_event` implementation differs from what I see in the codebase, or
2. The test is checking the wrong method

Consider verifying this behavior matches the actual `log_auth_event` implementation.